### DISCUSSION TITLE
microphone stream doesn't contain any supported code

### DISCUSSION THREAD

**Discussion by calvinchan999** [bluenviron/mediamtx#2737] [Questions]
## Which version are you using?

v1.2.0

## Which operating system are you using?

- [ ] Linux amd64 standard
- [ ] Linux amd64 Docker
- [ ] Linux arm64 standard
- [ ] Linux arm64 Docker
- [ ] Linux arm7 standard
- [ ] Linux arm7 Docker
- [ ] Linux arm6 standard
- [ ] Linux arm6 Docker
- [x] Windows amd64 standard
- [ ] Windows amd64 Docker (WSL backend)
- [ ] macOS amd64 standard
- [ ] macOS amd64 Docker
- [ ] Other (please describe)

## Describe the issue

Description

## Describe how to replicate the issue
I have successfully published microphone audio to the MediaMTX server through FFmpeg, and when I accessed the WebRTC, the MediaMTX server returned the error: 'closed: the stream doesn't contain any supported codec, which are currently AV1, VP9, VP8, H264, Opus, G722, G711.

1. start the server
2. publish audio source with this ffmpeg commend
`ffmpeg -i audio="Microphone (Yeti Stereo Microphone)" -c:a aac -c:v h264 -ar 44100 -b:a 1M -ac 1 -f rtsp rtsp://localhost:8554/test-1`

## Did you attach the server logs?

2023/11/22 12:59:31 INF MediaMTX v1.2.0
2023/11/22 12:59:31 INF configuration loaded from C:\Users\TEST\Desktop\mediamtx_v1.2.0_windows_amd64\mediamtx.yml
2023/11/22 12:59:31 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
2023/11/22 12:59:31 INF [RTMP] listener opened on :1935
2023/11/22 12:59:31 INF [HLS] listener opened on :8888
2023/11/22 12:59:31 INF [WebRTC] listener opened on :8889 (HTTP)
2023/11/22 12:59:31 INF [SRT] listener opened on :8890 (UDP)
2023/11/22 12:59:31 INF [API] listener opened on 127.0.0.1:9997
2023/11/22 12:59:55 INF [RTSP] [conn [::1]:60389] opened
2023/11/22 12:59:55 INF [RTSP] [session 46f54d3e] created by [::1]:60389
2023/11/22 12:59:55 INF [RTSP] [session 46f54d3e] is publishing to path 'test-1', 1 track (MPEG-4 Audio)
2023/11/22 13:03:45 INF [WebRTC] [session 8aade405] created by [::1]:56406
2023/11/22 13:03:45 INF [WebRTC] [session 8aade405] closed: the stream doesn't contain any supported codec, which are currently AV1, VP9, VP8, H264, Opus, G722, G711

## Did you attach a network dump?

no

**Comment by aler9** [comment]
Hello, as the error message suggests, use Opus instead of AAC:

```
ffmpeg -i audio="Microphone (Yeti Stereo Microphone)" -c:a libopus -b:a 64k -ar 44100 -ac 1 -f rtsp rtsp://localhost:8554/test-1
```

[a comment by an automated account is not shown]
